assets/javascripts/kitten/components/uploaders/image-drop-uploader/index.js

```diff
@@ -141,6 +141,9 @@ const StyledImageDropUploader = styled.div`
     }
   }
   .k-ImageDropUploader__manager__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
   }
```

Comment:
Je pense qu'on peut être plus concis comme ça, en utilisant directement le positionnement au sein de la flexbox :

```suggestion
  .k-ImageDropUploader__manager__content {
    align-self: center;
  }
```